test/gtest/common/test_ucc.h

```diff
@@ -206,9 +206,9 @@ class UccJob {
         UCC_JOB_CTX_GLOBAL, /*< ucc ctx create with OOB */
         UCC_JOB_CTX_GLOBAL_ONESIDED
     } ucc_job_ctx_mode_t;
-    static const int nStaticTeams     = 3;
+    static const int nStaticTeams     = 4;
     static const int staticUccJobSize = 16;
-    static constexpr int staticTeamSizes[nStaticTeams] = {2, 11, staticUccJobSize};
+    static constexpr int staticTeamSizes[nStaticTeams] = {1, 2, 11, staticUccJobSize};
```

Comment:
* We need a gtest to create team of size 1 from LOCAL context type (similar to test_team.cc, team_create_destroy_ctx_local) - just size 1
* lets also add test for context size 1 (global) creation into gtest/core/test_context; just to make sure it does not crash. Looks like if we allow team_size, then context size 1 is also reasonable.